### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.txt -L acces` and `typos --hidden --format brief`

### CODE REVIEW

File: tests/Commit.py [hunk 1]
```diff
@@ -132,12 +132,12 @@ def testCreateStatusWithAllParameters(self):
...
             "success",
             "https://github.com/jacquev6/PyGithub/issues/67",
-            "Status successfuly created by PyGithub",
+            "Status successfully created by PyGithub",
         )
         self.assertEqual(status.id, 277040)
         self.assertEqual(status.state, "success")
         self.assertEqual(status.target_url, "https://github.com/jacquev6/PyGithub/issues/67")
-        self.assertEqual(status.description, "Status successfuly created by PyGithub")
+        self.assertEqual(status.description, "Status successfully created by PyGithub")
```
Comment (EnricoMi) [inline tests/Commit.py:140]: this change requires changing `tests/ReplayData/Commit.testCreateStatusWithAllParameters.txt` as well.

File: tests/CommitStatus.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def testAttributes(self):
             datetime(2012, 9, 8, 11, 30, 56, tzinfo=timezone.utc),
         )
         self.assertEqual(self.statuses[0].creator.login, "jacquev6")
-        self.assertEqual(self.statuses[0].description, "Status successfuly created by PyGithub")
+        self.assertEqual(self.statuses[0].description, "Status successfully created by PyGithub")
```
Comment (EnricoMi) [inline tests/CommitStatus.py:59]: this change requires changing `tests/ReplayData/CommitStatus.testAttributes.txt` as well.

**Review by EnricoMi — CHANGES_REQUESTED** [review]
LGTM, some test fixes required.

**Review by billnapier — APPROVED** [review]

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**